### DISCUSSION TITLE
version 1.19.0

### DISCUSSION THREAD

**Discussion by basnijholt** [basnijholt/adaptive-lighting#735] [Announcements]
This is a massive release where we went through 26 PRs and 5 beta versions.

## Major Updates:

* New [Adaptive Lighting webapp with interactive graphs](https://basnijholt.github.io/adaptive-lighting/) (#719) @basnijholt
* Introduced `adapt_only_on_bare_turn_on` for immediate `manual_control` activation upon brightness or color changes during turn on (#709) @basnijholt
* Enabled simultaneous adaptation of lights instead of sequential adjustments (#529) @th3w1zard1
* Set parameters for `min_sunrise_time` and `max_sunset_time` (#707) @basnijholt
* Introduced various brightness ramping methods with `brightness_mode` (#699) @basnijholt
* Implemented call interception for handling multiple lights (#679) @basnijholt

## What’s Changed

* Reorganized option sequence in configuration flow (#700) @basnijholt
* Bump to 1.19.0 in manifest.json (#734) @basnijholt
* Fix length of descriptions in UI (config flow) (#732) @basnijholt
* Add release-drafter GitHub Action (#731) @basnijholt
* Fix adapt_only_on_bare_turn_on and apply does not result in manual_control (#729) @basnijholt
* Bump to 1.19.0b5 in manifest.json (#728) @basnijholt
* Prevent light.turn_on of light that was just turned off (#727) @basnijholt
* Mention app more prominently and improve style (#725) @basnijholt
* Split up SunLightSettings and improve https://basnijholt.github.io/adaptive-lighting/ (#719) @basnijholt
* [pre-commit.ci] pre-commit autoupdate (#722) @pre-commit-ci
* Bump to 1.19.0b4 in manifest.json (#718) @basnijholt
* Fix min_sunrise_time and max_sunset_time (#717) @basnijholt
* Small WebApp improvements (#716) @basnijholt
* Add simple webapp to play with different parameters (#715) @basnijholt
* Bump to 1.19.0b3 in manifest.json (#713) @basnijholt
* Fix adaptive_lighting.change_switch_settings service (#712) @basnijholt
* Bump to 1.19.0b2 in manifest.json (#710) @basnijholt
* Adapt lights simultaneously instead of one by one (#529) @th3w1zard1
* Add `adapt_only_on_bare_turn_on` which instantly triggers `manual_control` when turning on with brightness or color (#709) @basnijholt
* Add warning about Zigbee groups (#708) @basnijholt
* Add a min_sunrise_time and max_sunset_time (#707) @basnijholt
* Bump to 1.19.0b1 manifest.json (#706) @basnijholt
* Allow adaptive_lighting.apply to turn on lights and do not issue log message (#705) @basnijholt
* Logging fixes in significant_changes (#704) @basnijholt
* feat: add different brightness ramping mechanisms (#699) @basnijholt
* feat: change the order of options in config flow (#700) @basnijholt
* Implement call intercept for multiple lights (#679) @basnijholt

**Full Changelog**: https://github.com/basnijholt/adaptive-lighting/compare/1.18.3...v1.18.4

<hr /><em>This discussion was created from the release <a href='https://github.com/basnijholt/adaptive-lighting/releases/tag/1.19.0'>version 1.19.0</a>.</em>

**Comment by pqhf5kd** [comment]
I have several configuration options with missing info/descriptions after updating to 1.19.0

**Reply by basnijholt** [reply]
Probably the same as https://github.com/basnijholt/adaptive-lighting/issues/720. 

Likely it’s fixed by clearing browser cache.

**Reply by pqhf5kd** [reply]
Yes, sorry!

**Comment by jesse-kaufman** [comment]
I haven't been using HA all that long, but the new features and refinements in this plugin over just the last few months have been amazing! My circadian rhythm can't thank you enough!

**Reply by basnijholt** [reply]
Thanks for your kind words! Really appreciated 😄

**Comment by skycarl** [comment]
Incredible! This is such a fantastic addon, and the webapp is going to be super helpful.

**Reply by basnijholt** [reply]
Thank you for the kind words! Thrilled you're finding value in the addon and webapp. Your feedback means a lot!

**Comment by hastarin** [comment]
It seems biohacking is all the rage lately. I would not be surprised to see a Bryan Johnson video showing him using this addon for his house.

I know it certainly contributes to my quality of sleep.

Great work and thanks for all your hard work @basnijholt

**Reply by basnijholt** [reply]
I am happy to hear that!

I appreciate your positive feedback!

**Comment by FkoNL** [comment]
This is the reason I now have Home Assistant. Thanks for the hard work!

**Reply by basnijholt** [reply]
Thank you for the lovely compliment!

**Comment by Skorfulose** [comment]
A big „Thank You!“ from Germany! 😀

**Reply by basnijholt** [reply]
Grateful for your kind remarks 😄

**Comment by lencioni** [comment]
Thanks for all of your hard work on this project, I really appreciate it!

**Reply by basnijholt** [reply]
I am happy it is appreciated! 😄

**Comment by mindtripper** [comment]
Great release!
This is by far the most important custom component in my setup.
Thank you!

**Comment by dschoorisse** [comment]
Thank you! One of the main reasons for me too to start using Home Assistant for my lights :-)

**Comment by manuelh19** [comment]
Thank you for this amazing add on, the constant update, and your very fast, kind and helpful support! <3

**Comment by usma0118** [comment]
Thanks for good work :)

**Comment by andrewfitz** [comment]
Thank you!

**Comment by PhasorCZ** [comment]
Thank you for all the work on this!

**Comment by narfotic** [comment]
Thanks! Keep up the great work!

**Comment by karaminder** [comment]
Appreciate the hard work you guys do!

**Comment by filmgarage** [comment]
Great addition to the amazing Home Assistant!

**Reply by rkallensee** [reply]
I can't thank you enough for the fantastic work on this component! 👏🏼💡❤️

**Comment by synth1234** [comment]
Thanks for your hard work on this great Integration/Automation! keep up the awesome work.

**Comment by djurny** [comment]
After using f.lux for a long time, adaptive lighting was the first thing I was looking for after diving into Home Assistant. ...or was it the other way around?
In any case, goed bezig Bas!

**Comment by DTTerastar** [comment]
Keep on making home assistant better!

**Comment by tianzhuang55** [comment]
Just come to say thank you! Awesome project!

**Comment by Vertikar** [comment]
Thanks for all your hard work on this @basnijholt!!

**Comment by MattCheetham** [comment]
Thanks for all your hard work on this project!

**Comment by lennon101** [comment]
An incredible piece of work completed on this. It works really well for me. I just wanted to reach out and say thank you for the amazing job you did with this!

**Comment by matthiashh** [comment]
Thanks a lot. This project is one of the reasons for me to have HA

**Comment by bluemon82** [comment]
Really cool project ❤️
